content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -1,67 +1,236 @@
+   :align: center
+   :alt: View of the smart buttons on a ticket focusing on the coupon button.
+
+.. seealso::
+   `After-Sales Features <https://www.odoo.com/slides/slide/after-sales-features-1699>`_
+
+Facilitate a product return with a reverse transfer
+===================================================
+
+Returns are completed through *reverse transfers*, which generate new warehouse operations for the
+returning products. Click the :guilabel:`Return` button at the top left of a :guilabel:`Ticket` to
+open the :guilabel:`Reverse Transfer` pop-up window.
+
+.. image:: after_sales/after-sales-return-button.png
    :align: center
+   :alt: View of a Helpdesk ticket with the return button highlighted.
 
-Generate credit notes from tickets
-----------------------------------
+.. note::
+   The :guilabel:`Return` button will only appear on a :guilabel:`Ticket` if the customer selected
+   has recorded deliveries in the database.
 
-You can use a credit note to refund a customer or adjust the amount due. For that, simply go to
-your ticket page, click on *Refund* and select the corresponding *Invoice*. Clicking on *Reverse*
-generates a credit note, and you can *Post* it while still being in the *Helpdesk* app.
+By default, the quantity will match the validated quantity from the delivery order. Update the
+:guilabel:`Quantity` field if necessary.
 
-.. image:: after_sales/aftersales2.png
+.. image:: after_sales/after-sales-reverse-transfer.png
    :align: center
+   :alt: View of a reverse transfer creation page.
 
-Allow product returns from tickets
-----------------------------------
+Click :guilabel:`Return` to confirm the return. This generates a new warehouse operation for the
+incoming returned product(s). A :guilabel:`Returns` smart button will then be added to the top of
+the :menuselection:`Helpdesk` :guilabel:`Ticket`.
 
-The process of a product return from your customer back to your warehouse is taken into action when,
-at the ticket page, you choose the option *Return*.
+.. image:: after_sales/after-sales-return-smart-button.png
+   :align: center
+   :alt: View of the return smart button on a helpdesk ticket.
+
+.. seealso::
+   :doc:`/applications/sales/sales/products_prices/returns`
+
+Send products for repair from a ticket
+======================================
 
-.. image:: after_sales/aftersales3.png
+If the :guilabel:`Ticket` is related to an issue with a faulty or broken product, a repair order
+can be created from the :menuselection:`Helpdesk`, and managed through the :menuselection:`Repair`
+application.
+
+To create a new repair order, click :guilabel:`Repair`.
+
+.. image:: after_sales/after-sales-repair-reference.png
    :align: center
+   :alt: View of a repair reference page.
+
+If a product was specified in the :guilabel:`Product` field on the ticket, it will be automatically
+added to the :guilabel:`Product to Repair` field on the newly created :guilabel:`Repair Reference`.
```

Comment:
justify the tag: what is a **Repair Reference**? 
A form, a pop-up window, a form heading...? 🤔